ui/raidboss/data/06-ew/ultimate/the_omega_protocol.txt

```diff
@@ -199,6 +200,7 @@ hideall "--sync--"
 774.0 "Hyper Pulse" #sync / 1[56]:[^:]*:Right Arm Unit:7B71:/
 782.0 "Patch" #sync / 1[56]:[^:]*:Omega-M:7B63:/
 784.0 "Patch" #sync / 1[56]:[^:]*:Omega-M:7B63:/
+786.4 "Swivel Cannon" sync / 1[56]:[^:]*:Omega:7B94:/
```

Comment:
Probably there's two, for both directions? https://xivapi.com/search?indexes=action&string=Swivel%20Cannon&columns=Name,ID&pretty=true

Reply:
I suspect so. I did not know if we should confirm via log first or go ahead and put it in. I am fairly certain it can do both directions and that they would have two spells. Looks like it's a safe bet to place in 7B95 as the other.